seekr/console_scripts.py

```diff
@@ -614,7 +643,7 @@ def console_domain_pearson():
     parser.add_argument('-p', '--percentiles', help='Path to new csv file for storing percentiles.')
     parser.add_argument('-k', '--kmer', default=6,
                         help='Length of kmers you want to count.')
-    parser.add_argument('-nl', '--no_log2', action='store_false',
+    parser.add_argument('-l', '--log2', action='store_false',
```

Comment:
I'm losing a bit of steam because it's late and been a long day, but I'm not sure if this logic flip is right.

And even if it is correct, I don't know if it's ergonomic. The default for counting kmers is to log transform. But the default for pearson is now to not transform?